dojo-dungeon/src/dojo/rooms/builder/RoomBuilder.java

```diff
@@ -155,7 +155,7 @@ public Room buildRoom_L1_R3_Fragen_Lambda() {
    * @return the newly generated room instance
    */
   public Room buildRoom_L2_R1_Monster_2() {
```

Comment:
Dieser und weitere Methodennamen müssen noch umbenannt werden...

Reply:
... und auch noch innerhalb von RoomBuilder in die richtige Reihenfolge gebracht werden...